source/changelog/r91.inc

```diff
@@ -0,0 +1,37 @@
+.. _changelog-r91:
+
+*************************
+Release June 22nd 2021
+*************************
+The following changes started rolling out on Tuesday June 22nd 2021.
+The API and backend are being rolled out to all tenants at the same time, and the client will be propagated over the next couple of days.
+
+API and Backend
+===============
+The following new features and improvements are now available in the API and the backend.
+
+* Expanded the summary result information when importing Mastercard Paymentservices OCR files.
+* Adjustments to the BgMax file to handle payments from Autogiro.
+* Added API validation to prevent empty standalone account demands from being generated by billing.
+* Fixed an issue where reminder fees would not propagate correctly to the Invoice Document.
+* Fixed an issue where the reporting subsystem would not update payment agreements on subscriptions when automatically changed.
+* PayEx POPS related payments are now blocked entirely before being sent to PayEx (where they would sometime actually succeed even though the platform is reported shut down).
+* Fixed an issue where an attempt was made to automatically adjust payment agreements on cancelled subscriptions.
+* Fixed an issue with reminder fees not being accumulated on multiple reminders.
+* Fixed an issue with scheduling for Avtalegiro on the first subscription period would end up issuing demands with an adjusted due date but still keep Invoice as the provider type. Now it correctly keeps the original due date.
+* Fixed an issue where the existence of a contract fee would cause payment demands to not be generated.
+
+Merchant Client
+===============
+The following new features and improvements are now available in the merchant client.
+
+* Added option to credit an Enterprise Plan invoice directly from the invoice overview.
+* New styling, details and icons for payment agreement overview in merchant.
+* Multiple bug fixes and adjustments for temporary changes and temporary delivery stops **Preview**.
+* Fixed an issue that would display the wrong details when importing payments and mandates using Mastercard Paymentservices OCR file.
```

Comment:
This is not part of the current deployment, but I think the fix was done yesterday evening. So we might consider updating the client for second deployment batch.